extensions/ftp/src/main/java/org/mule/extension/ftp/internal/ftp/ClassicFtpFileAttributes.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.ftp.internal.ftp;
+
+import org.mule.extension.ftp.api.FtpFileAttributes;
+import org.mule.runtime.module.extension.file.api.AbstractFileAttributes;
+
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+
+import org.apache.commons.net.ftp.FTPFile;
+
+/**
+ * Implementation of {@link FtpFileAttributes} for files read from a FTP server.
+ *
+ * @since 4.0
+ */
+public class ClassicFtpFileAttributes extends AbstractFileAttributes implements FtpFileAttributes
```

Comment:
I meant normal, regular, good old fashioned FTP. That's the best name I could come out with, if you have a better idea I'm open to it